CMakeLists.txt

```diff
@@ -208,3 +208,33 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake/redis++.pc.in"
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake/redis++.pc"
         DESTINATION "lib/pkgconfig")
+
+option(REDIS_PLUS_PLUS_BUILD_DEBIAN "Build Debian package" OFF)
+
+if (REDIS_PLUS_PLUS_BUILD_DEBIAN)
+    SET(CPACK_GENERATOR "DEB")
+    SET(CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS "ON")
+
+    if (NOT DEFINED CPACK_DEBIAN_PACKAGE_MAINTAINER)
+        SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "anonymous")
+    endif()
+    if (NOT DEFINED CPACK_PACKAGE_NAME)
+        SET(CPACK_PACKAGE_NAME "libredis++-dev")
+    endif()
+    if (NOT DEFINED CPACK_DEBIAN_PACKAGE_DEPENDS)
+        SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libstdc++6, libhiredis-dev")
+    endif()
+
+    EXECUTE_PROCESS(COMMAND bash -c "dpkg --print-architecture" OUTPUT_VARIABLE CPACK_DEBIAN_PACKAGE_ARCHITECTURE)
+    string(REGEX REPLACE "\n$" "" CPACK_DEBIAN_PACKAGE_ARCHITECTURE "${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
+    SET(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}_${REDIS_PLUS_PLUS_VERSION}_${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
```

Comment:
It seems that we can simply call `set(CPACK_DEBIAN_FILE_NAME DEB-DEFAULT)`. No need to call `dpkg --print-architecture` to get related info.